src/Entity/TranslationRequest.php

```diff
@@ -228,6 +235,11 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setLabel('Data')
       ->setDescription(t('The Json encoded data to be translated.'));
 
+    $fields['log'] = BaseFieldDefinition::create('entity_reference')
```

Comment:
`logs`.

This needs to be multiple values since we will log multiple messages per translation request.